src/s3.zig

```diff
@@ -7,12 +7,47 @@ pub const RareData = @import("./bun.js/rare_data.zig");
 const JSC = bun.JSC;
 const strings = bun.strings;
 
+pub const ACL = enum {
+    // not informed means that the user did not specify an ACL
```

Comment:
Use `///` so it shows up in editor autocomplete

```suggestion
    /// The user did not specify an ACL
```